ccm_web/client/src/components/CustomTable.tsx

```diff
@@ -81,7 +81,7 @@ function CustomTable<T extends TableEntity> (props: TableProps<T>): JSX.Element
                 inputProps: { 'aria-label': 'rows per page' },
                 native: true
               }}
-              onChangePage={handleChangePage}
+              onPageChange={handleChangePage}

```

Comment:
Even though you have change the function name to `onPageChange` when i am clicking the pages button it is saying `onChangePage` is not a function. So did you changed in all places? [onChangePage](https://stackoverflow.com/a/67077268/724238)

Reply:
Good catch, I'll take a look.

Reply:
Okay, this should have fixed it, @pushyamig: https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/236/commits/12f2efaeaa3bff68618217b7f27a1bde7a4ab3de